lib/config.go

```diff
@@ -67,6 +66,11 @@ var DefaultCapabilities = []string{
 	"KILL",
 }
 
+// DefaultSysctls for the default-sysctls option in the crio.conf file
+var DefaultSysctls = []string{
+	"net.ipv4.ping_group_range=0 0",
```

Comment:
We cannot hard code this. For user namespaces, the gid_map could be different so will have to populate the range and validate it against user ns group mappings.

Reply:
I guess we need to see if these are relative to the USERNS.  IE Does this happen in runc after the USERNS is setup, then it should work.

Reply:
it should happen from inside the container once the user namespace is configured.  Some sysctl are not allowed from a namespace and the OCI runtime will fail to set these, but I don't think this PR has to worry of it.